recipe/conda_build_config.yaml

```diff
@@ -14,6 +14,8 @@ channel_sources:
 channel_targets:
   - conda-forge main
   - conda-forge python_debug   # [not win]
+is_python_min:
+  - no
```

Comment:
Could we put this next to the other `python` keys [from `zip_keys`]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/0c4535a2c15d4569f30a9395e2b93bdedf938ca4/recipe/conda_build_config.yaml#L186-L190 )? That would make it clearer this is related

https://github.com/conda-forge/python-feedstock/blob/37ffa1ad5b52acaf7cc2042e17641e5d421da514/recipe/conda_build_config.yaml#L1-L6

Reply:
Feel free to push into this PR if you feel strongly about it (same for #773).

Reply:
I know it sounds cosmetic, but there is a lot going on in this recipe (and this file). It helps a lot to understand the configurations here (especially for new contributors)